api/justfile

```diff
@@ -121,6 +121,11 @@ doc-test: wait-up
       --fail \
       'http://localhost:50280/v1/schema/' > openapi.json
 
+# Run the API schema tests using schemathesis
+test-schema: wait-up
+    just doc-test
+    python -m pip install schemathesis && st run --checks response_headers_conformance openapi.json --base-url http://localhost:50280 && st run --checks not_a_server_error openapi.json --base-url http://localhost:50280
```

Comment:
This step seems installs the `schemathesis` package on the host Python environment, which is not ideal. It's also not guaranteed to work on macOS for example, where there is no unversioned `python` (but `python3` is present across all platforms afaik).

<img width="258" alt="Screenshot 2023-12-11 at 7 43 24 AM" src="https://github.com/WordPress/openverse/assets/16580576/e8ecd408-04f2-46a0-b5f9-1440c0f446c2">

The [Getting Started](https://schemathesis.readthedocs.io/en/stable/getting-started.html#getting-started) guide lists a bunch of ways to run this, including a Docker image. That seems like a more cross-platform way to run the check that does not affect the host.